examples/stress_tests/bevymark.rs

```diff
@@ -98,45 +98,42 @@ fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
 
     commands.spawn_bundle(Camera2dBundle::default());
     commands
-        .spawn_bundle(
-            TextBundle::from_sections([
-                TextSection::new(
-                    "Bird Count: ",
-                    TextStyle {
-                        font: asset_server.load("fonts/FiraSans-Bold.ttf"),
-                        font_size: 40.0,
-                        color: Color::rgb(0.0, 1.0, 0.0),
-                    },
-                ),
-                TextSection::from_style(TextStyle {
+        .spawn_bundle(TextBundle::from_sections([
+            TextSection::new(
+                "Bird Count: ",
+                TextStyle {
                     font: asset_server.load("fonts/FiraSans-Bold.ttf"),
                     font_size: 40.0,
-                    color: Color::rgb(0.0, 1.0, 1.0),
-                }),
-                TextSection::new(
-                    "\nAverage FPS: ",
-                    TextStyle {
-                        font: asset_server.load("fonts/FiraSans-Bold.ttf"),
-                        font_size: 40.0,
-                        color: Color::rgb(0.0, 1.0, 0.0),
-                    },
-                ),
-                TextSection::from_style(TextStyle {
+                    color: Color::rgb(0.0, 1.0, 0.0),
```

Comment:
@alice-i-cecile Was this value changed deliberately?

Reply:
No; feel free to revert.